frontend/src/components/worksheets/BundleDetail/BundleActions.jsx

```diff
@@ -51,6 +51,13 @@ class BundleActions extends React.Component<
 	            >
 	            	Kill
 	            </Button>
+				<Button
```

Comment:
Locally it looks nice...
![Screenshot from 2019-11-30 10-07-52](https://user-images.githubusercontent.com/23012631/69904252-522e3b00-1359-11ea-86c3-3b4c514aad83.png)